src/sonic-host-services/scripts/hostcfgd

```diff
@@ -905,7 +931,7 @@ class NtpCfg(object):
             cmd = 'systemctl restart ntp-config'
             run_cmd(cmd)
 
-    def ntp_global_update(self, key, data):
+    def ntp_global_update(self, key, data, load=False):
```

Comment:
>load [](http://example.com/codeflow?start=43&length=4)

The original code (before any fix on blackout issue) is using `isLoad`.

If your intention is to return to original behavior, try use original code.

Reply:
The code is changed a bit so I think I have a bit of flexibility here and I am using that flexibility to try to make the naming convention a bit more consistent. 

If you dislike `load` how about `is_load`?

Reply:
I can close this one.